data_cleaning_job.ipynb

```diff
@@ -0,0 +1,2353 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #12.        state_df.to_csv(f'citizenData/{state_name}_citizenData.csv', index=False)
</pre><p>What is causing all of the errors in the output?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BU-Spark/pitne-season-watch/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
Sorry Colette, forgot to delete this notebook. This notebook is deprecated. I was trying to use this for the SCC but I found a workaround.